docs/development-environment.md

```diff
@@ -44,23 +48,23 @@ To get a local WordPress site up and running you need a web server (Apache, Ngin
 
 **Important:** Docker is the only solution that we recommend and can provide support for. The others are listed here as reference if you want to try something different. We won't be able to provide support for them
 
-* ### Docker (Supported Recommended)
+### Docker (Supported Recommended)
```

Comment:
So the bullets here worked ok because the contents of the sections were also part of the bullet. See https://github.com/Automattic/jetpack/blob/c81216cbcf03985541197001d42f3d04fdf2728b/docs/development-environment.md#docker-supported-recommended for example.

Now that you removed the bullets, the spaces at the starts of the sections are making them be displayed as `<pre>` blocks, which we don't want. Look at https://github.com/Automattic/jetpack/blob/f83a5b3f025556c265780ef9d36234036b505ecd/docs/development-environment.md#docker-supported-recommended for example.

If you want to go back and put bullets before your headings in your new section, it would probably work if you do like is done here to have the section contents themselves inside the bullets too.